src/icons/general/IconSecurityClassification.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import { bdlGray } from '../../styles/variables';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    strokeWidth?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconSecurityClassification = ({
+    className = '',
+    height = 13,
+    color = bdlGray,
+    title,
+    strokeWidth = 1,
+    width = 11,
+}: Props) => (
+    <AccessibleSVG
+        className={`bdl-IconSecurityClassification ${className}`}
+        height={height}
+        title={title}
+        viewBox="0 0 11 13"
```

Comment:
we should use an icon from design that has been exported at the appropriate viewbox size (32/32 or 16/16). With that, we can set the default props for width and height to be the default size (16x16 for badges, or 32x32 for our regular icons).

Reply:
Thanks @booc0mtaco 
The icon is updated, I leave the size as 10*10 since 16 looks a bit off here. 

<img width="229" alt="Screen Shot 2019-08-14 at 5 57 40 PM" src="https://user-images.githubusercontent.com/20669641/63066277-25dfd300-bebe-11e9-991a-ca80694d8c95.png">